eventkit_cloud/ui/static/ui/app/components/DataPackPage/LoadButtons.js

```diff
@@ -86,10 +90,10 @@ export class LoadButtons extends React.Component {
 
 LoadButtons.propTypes = {
     range: PropTypes.string.isRequired,
-    handleLoadLess: PropTypes.func.isRequired,
+    handleLoadLess: PropTypes.func,
```

Comment:
If a prop is not 'required' it should have a default value listed in Component.defaultProps. In this example that would be `LoadButtons.defaultProps = { . . . . . . };`

Reply:
I left that without a default prop since it's being checked whether it's passed in or not in the render function, so that if a handler isn't passed in then **Load Less** isn't shown at all. Do you think we should set a default of `null` just to be explicit about it?

Reply:
Oh, I should probably also clarify that I made that change so that I could reuse the LoadButtons component while only showing the **Load More** button.

Reply:
Yea I think `null` would be fine. Its more a documentation and linting thing really, functionally it doesn't matter.

Reply:
Ok cool. I'll take care of that. There might be one or two more areas where I set the props up this way. I'll check them all to make sure they're all explicitly set to null by default.

Reply:
What do you think we should do for props that should be undefined if they aren't passed in? For example, in some places there are function props that are received and then passed along down to child components, where the child's default will be ignored if the parent prop defaults to `null`. Do you think we should explicitly default those sorts of props to `undefined`, just for the sake of clarity?